dolphin/drivers/dellemc_storage/alert_handler.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import date
+
+from oslo_log import log
+
+LOG = log.getLogger(__name__)
+
+
+class AlertHandler(object):
+    """Alert handling functions for vmax driver"""
+    default_category = 'New'
+
+    def __init__(self):
+        pass
+
+    def add_trap_config(self, context, storage_id, trap_config):
+        """Config the trap receiver in storage system."""
+        pass
+
+    def remove_trap_config(self, context, storage_id, trap_config):
+        """Remove trap receiver configuration from storage system."""
+        pass
+
+    def parse_alert(self, context, alert):
+        """Parse alert data got from alert manager and fill the alert model."""
```

Comment:
Please define alert model first, and for those required filed, driver have to fill it even if the input alert does not have that field.

Reply:
can we also show alert model structure sample here .. ?